src/main/java/org/gridsuite/modification/server/entities/ElementaryModificationEntity.java

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities;
+
+import javax.persistence.*;
+
+import java.util.Set;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.ElementaryModificationInfos;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "elementaryModification")
+public class ElementaryModificationEntity extends ModificationEntity {
+    @Column(name = "equipmentId")
+    private String equipmentId;
+
+    @Transient
```

Comment:
why Transient ? If we don't want to store it, but still return it to the http request creating the modification, then we should refactor the code. It's not simple to remember that some fields are valid only when the entity is created, not when it is loaded.
Or do we want to persist this information ?

Reply:
Exact
I'm going to return only these information for the http request

Reply:
Done